recipe/conda_build_config.yaml

```diff
@@ -1,2 +1,4 @@
 openblas:
   - 0.3.20
```

Comment:
Could you remove openblas because of https://github.com/AnacondaRecipes/aggregate/blob/481a058495037b14269d10f73411b374c2537a78/conda_build_config.yaml#L164 ?

Reply:
Done! Ty